src/libraries/Microsoft.PowerFx.Core/Syntax/UserDefinitions.cs

```diff
@@ -98,7 +96,7 @@ private IEnumerable<UserDefinedFunction> CreateUserDefinedFunctions(IEnumerable<
             foreach (var udf in uDFs)
             {
                 var udfName = udf.Ident.Name;
-                if (_restrictedUDFNames.Contains(udfName) || texlFunctionSet.AnyWithName(udfName) || BuiltinFunctionsCore._library.AnyWithName(udfName) || BuiltinFunctionsCore.OtherKnownFunctions.Contains(udfName))
+                if (_restrictedUDFNames.Contains(udfName) || texlFunctionSet.AnyWithName(udfName) || BuiltinFunctionsCore.IsKnownPublicFunction(udfName))
```

Comment:
>BuiltinFunctionsCore.IsKnownPublicFunction(udfName) [](http://example.com/codeflow?start=101&length=51)

@VamsiModem Now this is also checking `_featureGateFunctions`. Do you see a potential problem?